src/platform/Linux/BUILD.gn

```diff
@@ -182,4 +185,37 @@ static_library("Linux") {
   if (chip_device_config_enable_wifipaf) {
     public_deps += [ "${chip_root}/src/wifipaf" ]
   }
+
+  include_dirs = [
+    "${chip_root}/third_party/hostap/repo/src/common",
+  ]
+
+  if (chip_device_config_enable_wifipaf_ctrl_iface) {
+    sources += [
+        "WiFiPAFDriverCtrlIface.cpp",
+        "WiFiPAFDriverCtrlIface.h"
+    ]
+
+    deps += [ ":wpa_ctrl" ]
+  }
+  else {
+    sources += [
+        "WiFiPAFDriverDbus.cpp",
+        "WiFiPAFDriverDbus.h"
+    ]
+  }
+}
+
+source_set("wpa_ctrl") {
```

Comment:
this should be inside the hostap BUILD.gn, so that I can depend on hostap and not have a separate source set here.